bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/messages/Messages.properties

```
@@ -0,0 +1,90 @@
+
+mvn_int_test_report_no_project_found=An error was detected when the view integration test report request was processed. The object that represents the selected project was not found. When you use the Run Configuration launcher, be sure to select a project or project content first.
+mvn_int_test_report_none_found=No integration test results were found for project {0}. Select "{1}" before you select "{2}" on the menu.
+mvn_int_test_report_general_error="An error was detected when the view integration test report request was processed on project {0}.
+
+mvn_unit_test_report_no_project_found=An error was detected when the view unit test report request was processed. The object representing the selected project could not be found. When using the Run Configuration launcher, be sure to select a project or project content first.
+mvn_unit_test_report_none_found=No unit test results were found for project {0}. Select "{1}" before you select "{2}" on the menu.
+mvn_unit_test_report_general_error=An error was detected when the view unit test report request was processed on project {0}.
+
+gradle_test_report_no_project_found=An error was detected when the view test report request was processed. The object representing the selected project could not be found. When using the Run Configuration launcher, be sure to select a project or project content first.
+gradle_test_report_none_found=No test results were found for project {0}. Select "{1}" before you select "{2}" on the menu.
+gradle_test_report_general_error=An error was detected when the view test report request was processed on project {0}.
+
+issue_stop_prompt=Would you like to issue the Liberty Maven or Gradle stop command for this project to stop a Liberty server that may still be running the project outside of the Liberty Tools session?
+plugin_stop_issue_error=Exception issuing plugin stop command.
+plugin_stop_timeout=The Liberty Maven or Gradle stop command issued for project {0} timed out after {1} seconds.
+plugin_stop_failed=Stop failed with exitValue = {0}
+plugin_stop_general_error=An error was detected while processing the Liberty Maven or Gradle stop command on project {0}.
+
+# WorkspaceProjectsModel
+project_analyze_error=An error occurred when the projects in the workspace were analyzed.
+
+# DashboardView
+project_not_gradle_or_maven=Project {0} is not a Gradle or Maven project.
+image_descriptions_error=An error was detected while retrieving image descriptions.
+action_general_error=An error was detected during the {0} action.
+dashboard_refresh_error=An error was detected when the Liberty dashboard content was refreshed.
+
+# JRETab
+java_default_set_error=Unable to set the default Java installation that was obtained from the build path of project {0} in configuration {1}.
+java_resolution_error=Unable to resolve the Java installation path by using configuration {0}. Using the workspace Java installation.
```

```suggestion
java_resolution_error=Unable to resolve the Java installation path by using the {0}. Using the workspace Java installation instead.
```

What entity is Using the workspace Java installation instead? adda subject to the sentence if possible